Add option to copy absolute path in file system dock popup

Krigu opened the pull request:
This is the implementation of the proposal [7521 - Add "Copy Absolute Path" to filesystem folders ](https://github.com/godotengine/godot-proposals/issues/7521)  

The PR adds a new entry in the file system popup which adds the option to copy the absolute path of a folder or a file. This simplifies the workflow when opening assets from godot projects with third-party apps.

![screenshot_copy_absolute_path](https://github.com/godotengine/godot/assets/1186203/d7366c10-4d09-4fe6-8217-b2e1041ef196)

* *Bugsquad edit, closes: https://github.com/godotengine/godot-proposals/issues/7521*

Krigu committed:
Add option to copy absolute path in file system dock popup

timothyqiu commented:
I prefer not setting an icon for this menu item.

- So that this menu feels less crowded.
- It's the same as "Copy Path", but much less used.

Krigu commented:
I updated the PR and removed the icon as suggested by @timothyqiu

![copy_absolute_path_no_icon](https://github.com/godotengine/godot/assets/1186203/54436f29-3844-4ebb-90bf-cab12b87b293)

akien-mga commented:
Thanks! And congrats for your first merged Godot contribution :tada:
